docs/getting-started/endgame-sensor-FDA.asciidoc

```diff
@@ -45,15 +50,17 @@ image::images/fda/allow-kernel-ext.png[Allow kernel extension]
 If the prompt does not appear, enable the extension by doing the following:
 
 . Open a Terminal application.
-. Enter `kextload /Library/Extension/kendpoint.kext`. Prepend the command with `sudo` if necessary.
-You should receive an output similar to `149    0 0xffffff7f82e7b000 0x21000    0x21000    co.elastic.kendpoint (7.11.0) BD152A57-ABD3-370A-BBE8-D15A0FCBD19A <6 5 2 1>`. If you receive this output, the kernel extension is enabled.
-
+. Enter `kextload /Library/Extension/kendpoint.kext`. Prepend the command with `sudo` if necessary. You will receive an output that's similar to the one below:
++
+`149    0 0xffffff7f82e7b000 0x21000    0x21000    co.elastic.kendpoint (7.11.0) BD152A57-ABD3-370A-BBE8-D15A0FCBD19A <6 5 2 1>`
++
+If you receive this output, the kernel extension is enabled.
```

Comment:
Add content from the "Approve network content filtering for Endgame" [section](https://security-docs_3565.docs-preview.app.elstc.co/guide/en/security/master/endgame-sensor-full-disk-access-ven.html#allow-network-filter-content-ven). Also include caveat that these instructions only apply for users who are on 11+.

Reply:
Addressed in [e79a88e](https://github.com/elastic/security-docs/pull/3565/commits/e79a88eae0769a264e5714fd880e2ea81ed85965) and [7115d24](https://github.com/elastic/security-docs/pull/3565/commits/7115d2435376b99a5d3ff4666bda3709138d5762)